coding/javascript.md

```diff
@@ -29,6 +29,12 @@ e.g. `<a data-js="index-link" href="/index">Index</a>`.
 
 ## Tooling
 
+### Build tools
+[Webpack](https://webpack.js.org/) is the preferred build tool for applications, and [Rollup](https://rollupjs.org/guide/en) has generally been preferred for building
```

Comment:
Would it be okay if we make the language a little more opinionated here, to make our position clearer for people to use?

I'm not quite sure where we landed on the advice - are we really only recommending Rollup for libraries where "cross environment" libraries with "ES6 modules"? It would be much neater if we could simply say - Webpack for applications, Rollup for libraries / packages. Can that be our position?

If so, maybe the language could simply be:

> Webpack is the recommended build tool for applications, Rollup is recommended for packages.
> 
> Rollup and Webpack for the most part have feature parity now however (Webpack has support for scope hoisting, and Rollup supports code splitting), so this is mostly a matter of taste and convenience, given Rollup is easier to configure for library builds.

Thoughts?

Reply:
wfm, thanks @nottrobin